<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NuevaActuacion</class>
 <widget class="QDialog" name="NuevaActuacion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>209</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Nueva actuaci贸n</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Ingrese los datos de la nueva actuaci贸n</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::ExpandingFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="lblDescripcion">
          <property name="text">
           <string>Descripci贸n</string>
          </property>
          <property name="buddy">
           <cstring>txtDescripcion</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="txtDescripcion"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lblJuzgado_2">
          <property name="text">
           <string>Juzgado:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="lblJuzgado">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::ActionsContextMenu</enum>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lblFecha">
          <property name="text">
           <string>Fecha:</string>
          </property>
          <property name="buddy">
           <cstring>dteFecha</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDateTimeEdit" name="dteFecha">
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="lblFechaProxima">
          <property name="text">
           <string>Fecha pr贸xima:</string>
          </property>
          <property name="buddy">
           <cstring>dteFechaProxima</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDateTimeEdit" name="dteFechaProxima">
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <widget class="QCheckBox" name="checkCita">
          <property name="text">
           <string>Cita</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnAdd">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NuevaActuacion</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NuevaActuacion</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
